app/keepers.go

```diff
@@ -274,15 +274,16 @@ func (app *OsmosisApp) InitNormalKeepers(
 	// Create Transfer Keepers
 	transferKeeper := ibctransferkeeper.NewKeeper(
 		appCodec, keys[ibctransfertypes.StoreKey], app.GetSubspace(ibctransfertypes.ModuleName),
-		app.IBCKeeper.ChannelKeeper, &app.IBCKeeper.PortKeeper,
+		app.IBCKeeper.ChannelKeeper, app.IBCKeeper.ChannelKeeper, &app.IBCKeeper.PortKeeper,
```

Comment:
https://github.com/cosmos/ibc-go/blob/v3.0.0/docs/migrations/v2-to-v3.md for context